composer.json

```diff
@@ -18,9 +18,11 @@
     "require": {
         "php": ">=5.5.9",
         "doctrine/common": "~2.4",
+        "fig/link-util": "^1.0",
```

Comment:
We need this package in the Twig extension and the FrameworkBundle  (even at runtime). But the those packages have only a soft dependency to WebLink. In the context of the full stack, I think it makes sense to add it to have something working out of the box.